deeds-dapp-service/src/main/java/io/meeds/deeds/service/MeedTokenMetricService.java

```diff
@@ -172,6 +188,14 @@ public Map<String, BigDecimal> getLockedBalances() {
     return lockedBalances;
   }
 
+  public BigDecimal getTotalLockedBalance(Map<String, BigDecimal> lockedBalances) {
```

Comment:
```suggestion
  private BigDecimal getTotalLockedBalance(Map<String, BigDecimal> lockedBalances) {
```
utility internal method, not for Service API